lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -125,6 +155,19 @@ export class NativeInteractionClient extends BaseInteractionClient {
             provider,
             correlationId
         );
+        this.skus = setSKUs(this.nativeMessageHandler);
+    }
+
+    /**
+     * Adds SKUs to request extra query parameters
+     * @param request {NativeTokenRequest}
+     * @private
+     */
+    private addRequestSKUs(request: NativeTokenRequest) {
+        request.extraParameters = {
+            ...request.extraParameters,
+            "x-client-xtra-sku": this.skus,
```

Comment:
Please add as constant to [list of acceptable params](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/constants/AADServerParamKeys.ts)